qgroundcontrol.qrc

```diff
@@ -247,13 +247,11 @@
     <qresource prefix="/qml">
         <file alias="test.qml">src/test.qml</file>
         <file alias="QmlTest.qml">src/QmlControls/QmlTest.qml</file>
-
```

Comment:
Any reason to pull out these spaces? I put them in there to help with grouping. Makes it easier to read.